wordpress-mobile/WordPress-Android
# Jetpack Kotlinization: JetpackSecuritySettingsActivity

Parent #14904

This PR converts the Java `JetpackSecuritySettingsActivity.java` file to the Kotlin `JetpackSecuritySettingsActivity.kt` file.

To test:
- Launch the app.
- Login with a site that has Jetpack installed and activated (I used a `Pressable` site).
- Navigate to `My Site` -> `Jetpack Settings`.
- The `Security` Jetpack settings screen should be shown (see `after` screenshot below).
- Interact with this screen and verify that everything works as expected.
  - Clicking or swapping back should navigate you back.
  - Switching on/off the individual settings should work as before.

Note that as part of this update, the first `Feature` commit actually updated the toolbar back icon from the previous `close` icon to the default `back` icon. This was done because all other screens, including the `Site Settings` and `iOS` related screens are using the default `back` icon as well. Also, this `close` drawable was including from within a Gutenberg library, which indicates that something was not right. See the before and after below:

Before | After
-----|-----
<img width="250" height="500" alt="before" src="https://user-images.githubusercontent.com/9729923/123446615-bd95c880-d5e1-11eb-83b9-c79944e28cc8.png"> | <img width="250" height="500" alt="after" src="https://user-images.githubusercontent.com/9729923/123446625-bff82280-d5e1-11eb-859f-f6253b5b4176.png">

## Regression Notes
1. Potential unintended areas of impact

N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

Manually tested as described above in the `To test` section.

3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
